### PR TITLE
Update Remote Settings DEV URL

### PR DESCRIPTION
**Pull request by leplatrem**

### CODE REVIEW

**Comment by grahamalama** [comment]
@gcoxmoz https://mozilla-hub.atlassian.net/browse/OPST-962

**Review by gcoxmoz — APPROVED** [review]
Tweaking an existing integration, ownership seems right, we have prior art of using allizom in this config.  LGTM.

File: apps.yml [hunk 1]
```diff
@@ -3638,7 +3638,7 @@ apps:
     logo: auth0.png
     name: settings.dev.mozaws.net
```
Comment (gcoxmoz) [inline apps.yml:3639]: ugh.  I click approve right as I see something.
Should the name line change to match the URL?

**Comment by leplatrem** [comment]
@gcoxmoz this should be ok now

**Pull request merged**